### ISSUE TITLE
Error when loading embeddings model from local file...

### ISSUE THREAD

**Issue by oterrier** [nkthiebaut/zeugma#27]
If the file path contains upper case characters then it is not recognized because the `EmbeddingTransformer` `__init__` method does a conversion to lower case before testing the `os.path.exists(model)` 

Best regards

Olivier Terrier

**Comment by nkthiebaut** [comment]
Good catch, we could move the `os.path.exists(model)` test up to fix this issue. I'll fix it if I have time, but feel free to raise a PR if you do first.
